src/locales/de/Export.json

```diff
@@ -1,14 +1,15 @@
 {
     "Content": {
         "Title": "Daten exportieren",
-        "Text": "Sie können wählen, ob der komplette Datensatz, die angezeigte Statistik oder beides herunterladen werden soll."
+        "Text": "Sie können wählen, ob die ausgewählten Einzeldaten, die angezeigten agregierten Daten, die grafische Darstellung oder alles herunterladen werden soll."

```

Comment:
Typo: aggregierten
I'm not sure, but i think "alles" should start with a capital letter.

Reply:
Done.